go/client/chat_api_handler.go

```diff
@@ -99,15 +99,38 @@ type ChatAPI struct {
 
 // ChatChannel represents a channel through which chat happens.
 type ChatChannel struct {
-	Name      string `json:"name"`
-	Public    bool   `json:"public"`
-	TopicType string `json:"topic_type,omitempty"`
-	TopicName string `json:"topic_name,omitempty"`
+	Name        string `json:"name"`
+	Public      bool   `json:"public"`
+	MembersType string `json:"members_type"`
+	TopicType   string `json:"topic_type,omitempty"`
+	TopicName   string `json:"topic_name,omitempty"`
 }
 
+var validMembersTypes = []string{"", "kbfs", "team"}
+
 // Valid returns true if the ChatChannel has at least a Name.
 func (c ChatChannel) Valid() bool {
-	return len(c.Name) > 0
+	if len(c.Name) == 0 {
+		return false
+	}
+	validTyp := false
+	for _, typ := range validMembersTypes {
+		if typ == c.MembersType {
+			validTyp = true
+			break
+		}
+	}
+	return validTyp
+}
+
+func (c ChatChannel) GetMembersType() chat1.ConversationMembersType {
+	switch c.MembersType {
```

Comment:
could you use `chat1.ConversationMembersTypeMap` here?  or you want to avoid IMPTEAM?